google/resource_logging_folder_sink.go

```diff
@@ -13,12 +14,17 @@ func resourceLoggingFolderSink() *schema.Resource {
 		Delete: resourceLoggingFolderSinkDelete,
 		Update: resourceLoggingFolderSinkUpdate,
 		Schema: resourceLoggingSinkSchema(),
+		Importer: &schema.ResourceImporter{
+			State: resourceLoggingSinkImportState("folder"),
+		},
 	}
 	schm.Schema["folder"] = &schema.Schema{
-		Type:             schema.TypeString,
-		Required:         true,
-		ForceNew:         true,
-		DiffSuppressFunc: optionalPrefixSuppress("folders/"),
+		Type:     schema.TypeString,
+		Required: true,
+		ForceNew: true,
+		StateFunc: func(v interface{}) string {
```

Comment:
Is a statefunc better than a diff suppress or flattener for this?

Reply:
Oops! Just saw this comment now. Like I said in the PR description, this is just some weirdness with   one of the tests - it wasn't following the diff suppress and wrongly errored out comparing `folder/1234` to `1234`, but it played nicely with the StateFunc so I used that instead.

Reply:
(I think this is a function of the importstateverify tests doing a direct comparison of what's in state, meaning diff suppress functions aren't used, but I could be wrong.)